MANIFEST.in

```diff
@@ -7,6 +7,9 @@ include src/integrator_hybrid.c
 include src/integrator.c
 include src/gravity.c
 include src/collision.c
+include src/communication_mpi.c
```

Comment:
I couldn’t tell why some were included and not others, and I think in part of my testing, I added them all to make sure they were all added to the tarball on pypi.  I think you can just remove them all if you don’t want them (except rebound/rebound.h)
> On Jan 26, 2016, at 5:09 PM, Hanno Rein <notifications@github.com> wrote:
> 
> In MANIFEST.in <https://github.com/hannorein/rebound/pull/120#discussion_r50908938>:
> 
> > @@ -7,6 +7,9 @@ include src/integrator_hybrid.c
> >  include src/integrator.c
> >  include src/gravity.c
> >  include src/collision.c
> > +include src/communication_mpi.c
> Do you really need those?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/hannorein/rebound/pull/120/files#r50908938>.
>